{{ cookiecutter.package_name }}/.github/workflows/ci_tests.yml

```diff
@@ -0,0 +1,84 @@
+# GitHub Actions workflow for testing and continuous integration.
+#
+# This file performs testing using tox and tox.ini to define and configure the test environments.
+
+name: CI Tests
+
+on:
+  push:
+    branches:
+    - master
+    tags:
+  pull_request:
```

Comment:
Might be too much of an edge case, but adding
```
  pull_request:
    branches:  # only build on PRs against main
      - master
```

likewise limits PR builds. I have this set on my private repos to preserve minutes (public too if I forgot to comment it out).

Reply:
Probably too specific to be useful. Most repos don't do branch and for those that do, they probably want to test even if a PR is opened against older release branch (especially if they are following gitflow).

Reply:
this actually reminds me that we should be changing `master` here to `main` which is GitHub's new default...